pandas/_libs/tslibs/conversion.pxd

```diff
@@ -33,3 +32,21 @@ cdef maybe_datetimelike_to_i8(object val)
 cdef int64_t tz_convert_utc_to_tzlocal(int64_t utc_val, tzinfo tz)
 
 cpdef datetime localize_pydatetime(datetime dt, object tz)
+
+cdef extern from "numpy/ndarraytypes.h":
```

Comment:
Not sure if there is a better way to do this declaration. Right now I copy/pasted in the three files where it is need but is there a shared place to put that for shared access across the modules?

Reply:
Since it is in np_datetime.pxd, it can be `cimport`ed from there.